cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -578,14 +664,16 @@ func (scaleSet *ScaleSet) setInstanceStatusByProviderID(providerID string, statu
 	scaleSet.lastInstanceRefresh = time.Now()
 }
 
-// instanceStatusFromVM converts the VM provisioning state to cloudprovider.InstanceStatus
-func instanceStatusFromVM(vm compute.VirtualMachineScaleSetVM) *cloudprovider.InstanceStatus {
-	if vm.ProvisioningState == nil {
+// instanceStatusFromProvisioningState converts the VM provisioning state to cloudprovider.InstanceStatus
+func instanceStatusFromProvisioningState(provisioningState *string) *cloudprovider.InstanceStatus {
+	if provisioningState == nil {
 		return nil
 	}
 
+	klog.V(4).Infof("Getting vm instance provisioning state %s", *provisioningState)
```

Comment:
Best bump to V(5) for per VM logs
```suggestion
	klog.V(5).Infof("Getting instance provisioning state %s", *provisioningState)
```

Reply:
Fixed